### PR TITLE
Track product and tax rates importer view triggered from WP importer/exporter

### LINKED ISSUE #50757: Add tracking to WordPress import/export screens
**Prerequisites (mark completed items with an [x]):**
- [ ] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [ ] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

**Issue Description:**

We would like to understand how many WooCommerce merchants access the WordPress inport/export screens under the Tools menu.

## Acceptance Criteria

Add tracks events for
* Views for Import screen (Tools > Import)
* Views for Export screen  (Tools > Export)
* Clicks on WooCommerce products (CSV) run importer
* Clicks on WooCommerce tax rates (CSV) run importer

### PR DESCRIPTION
**Pull request by moon0326**
…screen

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #50757 

This PR adds the following tracks 

**product_importer_view_from_wp_importer:** Triggered when a user clicks on `Run importer` for `WooCommerce products (CSV)` on the Tools -> Import page.

**tax_rates_importer_view_from_wp_importer:** Triggered when a user clicks on `Run importer` for `WooCommerce tax rates (CS)` on the Tools -> Import page.

**wordpress_export_view**: Triggered when a user views `Tools -> Export` page.
**wordpress_import_view**: Triggered when a user views `Tools -> Import` page.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch.
2. Make sure `Allow usage of WooCommerce to be tracked` is checked on `WooCommerce -> Settings -> Advanced -> WooCommerce.com`
3. Open your browser's inspector and click on the `Network` tab and search for `t.gif`
4. Navigate to `Tools -> Export`.
5. Confirm the `wordpress_export_view` event.
6. Navigate to `Tools -> Import`.
7. Confirm the `wordpress_import_view` event.
8. Click on `Run Importer` for `WooCommerce products (CSV)`
9. Confirm the `product_importer_view_from_wp_importer` event.
10. Go back to the importer screen.
11. Click on `Run Importer` for `WooCommerce tax rates (CSV)`
12. Confirm the `tax_rates_importer_view_from_wp_importer` event.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [x] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

Add tracks for WordPress Importer/Export pages.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/includes/admin/class-wc-admin-importers.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+<?php //phpcs:ignore Generic.PHP.RequireStrictTypes.MissingDeclaration
```
Comment (moon0326) [inline plugins/woocommerce/includes/admin/class-wc-admin-importers.php:1]: This is an ancient file and I think it's best to keep it as it is for backward compatibility. 

This is a warning for not having `declare(strict_types=1);`

**Comment by psealock** [comment]
@moon0326 I'm seeing a white screen on the Profiler on JN but its not happening locally nor the trunk branch on JN.

**Comment by moon0326** [comment]
> Use $this for the callback

Good catch 👍 Pushed [Use $this for the callback](https://github.com/woocommerce/woocommerce/pull/50769/commits/e76ae2496ea6a69a472e3e5a63a8738596dc81f9) and it tested well on a JN site. Could you take a look at it again please?

**Review by psealock — APPROVED** [review]
This is testing well and code changes look good too 🚢

**Review by ilyasfoo — COMMENTED** [review]
@moon0326 Tests well according to instructions but I'm seeing `wcadmin_wordpress_import_view` track being fired even in blogger import. I tried this by:

1. Go to Tools > Import
2. Click `Install now` under Blogger
3. Click `Run importer`
4. Observe the view track being fired in the screen

Is this expected?

**Comment by moon0326** [comment]
> @moon0326 Tests well according to instructions but I'm seeing `wcadmin_wordpress_import_view` track being fired even in blogger import. I tried this by:
> 
> 1. Go to Tools > Import
> 2. Click `Install now` under Blogger
> 3. Click `Run importer`
> 4. Observe the view track being fired in the screen
> 
> Is this expected?

Good catch! Fixed it in [Do not track if we are in a specific import screen](https://github.com/woocommerce/woocommerce/pull/50769/commits/a0e6351bcced9891f0d314ce38afcea8297aff66)

Could you please take a look at it again 🙏 ?

**Review by ilyasfoo — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**